R/findNLDI.R

```diff
@@ -76,21 +74,25 @@ get_nldi_sources <- function(url = pkg.env$nldi_base) {
 #' @examplesIf is_dataRetrieval_user()
 #' \donttest{
 #' base <- "https://api.water.usgs.gov/nldi/linked-data/"
-#' get_nldi(paste0(base, "comid/101"), type = "feature", use_sf = FALSE)
-#' get_nldi(paste0(base, "comid/101"), type = "feature", use_sf = TRUE)
-#' get_nldi(url = paste0(base, "nwissite/USGS-11120000"), type = "feature", use_sf = TRUE)
-#' get_nldi(paste0(base, "nwissite/USGS-11120000"), type = "feature", use_sf = TRUE)
+#' dataRetrieval:::get_nldi(paste0(base, "comid/101"), type = "feature", use_sf = FALSE)
```

Comment:
I think CRAN gets mad at `:::` even if it's never actually run -- believe this gets caught by their automated checks. Maybe not an issue due to `@noRd`, but I'm not 100% sure.

If it is an issue, the workaround is to use `getFromNamespace()` to assign the non-exported object into the current environment, which defeats the static analysis checks. (If it's not an issue with CRAN, well, sorry for the distraction!)